vortex-array/src/array/chunked/stats.rs

```diff
@@ -14,9 +14,10 @@ impl ArrayStatisticsCompute for ChunkedArray<'_> {
                 s.compute(stat);
                 s.to_set()
             })
-            .fold(StatsSet::new(), |mut acc, x| {
```

Comment:
Is this addressed by #303 ?

Reply:
303 is a superset of this change that also fixes some other issues